components/CreateOrganizationForm.js

```diff
@@ -1,289 +1,421 @@
+  };
 
-            .image .InputTypeDropzone {
-              width: 100px;
-            }
+  const removeAdmin = collective => {
+    const filteredAdmins = admins.filter(admin => admin.member.id !== collective.id);
+    setAdmins(filteredAdmins);
+    updateAdmins(admins);
+  };
 
-            .backgroundImage-dropzone {
-              max-width: 500px;
-              overflow: hidden;
-            }
+  // Update admins whenever there is a change
+  useEffect(() => {
+    if (admins.length) {
+      updateAdmins(admins);
+    }
+  }, [admins]);
 
-            .user .image-dropzone {
-              width: 64px;
-              height: 64px;
-              border-radius: 50%;
-              overflow: hidden;
+  return (
+    <Flex flexDirection="column">
+      <Flex flexDirection="column" my={[0, 2]}>
+        <Box>
+          <BackButton asLink onClick={() => window && window.history.back()}>
+            ←&nbsp;
+            <FormattedMessage id="Back" defaultMessage="Back" />
+          </BackButton>
+        </Box>
+        <Box mb={[1, 3]} alignItems="center">
+          <H1
+            width={[288, 264]}
+            fontSize="28px"
+            lineHeight="36px"
+            fontWeight="500"
+            textAlign={['left', 'left', 'left', 'center']}
+            color="black.900"
+            letterSpacing="1px"
+            mt={2}
+            px={[2, 4, 0]}
+          >
+            <FormattedMessage id="create.org.title" defaultMessage="Create Organization" />
+          </H1>
+        </Box>
+      </Flex>
+      {error && (
+        <Flex alignItems="center" justifyContent="center">
+          <MessageBox type="error" withIcon mb={[1, 3]} data-cy="cof-error-message">
+            {error}
+          </MessageBox>
+        </Flex>
+      )}
+      <Formik validate={validate} initialValues={initialValues} onSubmit={submit} validateOnChange={true}>
+        {formik => {
+          const { values, handleSubmit, errors, touched, setFieldValue } = formik;
+          const suggestedSlug = value => {
+            const slugOptions = {
+              replacement: '-',
+              lower: true,
+              strict: true,
+            };
+            return trim(slugify(value, slugOptions), '-');
+          };
+          const handleSlugChange = e => {
+            if (!touched.slug) {
+              setFieldValue('slug', suggestedSlug(e.target.value));
             }
+          };
+          return (
+            <Form>
+              <Container flexDirection="column" justifyContent="center" px={[1, 30, 150]} mt={4}>
+                <Container display="flex" flexDirection={['column', 'row', 'row']}>
+                  <Flex
+                    flexDirection="column"
+                    mx={[1, 10, 10]}
+                    width={[320, 350, 376, 472]}
+                    justifyContent="space-around"
+                  >
+                    <H4 color="black.900" fontSize="28px">
```

Comment:
This is the correct font size for tablet, but on desktop the font for this and "Administrators" title is `18px`. 

https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/OC.com-Production-Ready-%E2%9C%85?node-id=2375%3A2